platform/metrics/platform-metrics-webconsole-plugin/pom.xml

```diff
@@ -1,141 +0,0 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
I believe it was specific to the Felix webconsole, I performed some ingests and queries and they showed up on the admin ui metrics page.